### PR TITLE
Update USAGE.md

### PR DESCRIPTION
**Pull request by nickforsythbarr**
issues and workarounds readme section added with specific reference to: `ERROR:nsNSSCertificateDB.cpp(95)] PK11_ImportCert failed with error -8168` and the solutions to that.

### CODE REVIEW

**Review by pi-314159 — APPROVED** [review]

File: chromium/USAGE.md [hunk 1]
```diff
@@ -17,4 +17,27 @@ Please note that not all algorithm combinations are expected to work. Most notab
...
+### Ubuntu/Deb
+
+`ERROR:nsNSSCertificateDB.cpp(95)] PK11_ImportCert failed with error -8168`
+
+There are a few things to try should you recieve this error whilst importing your proxy CA certificate:
+
+1. Manually install CA cert into nssdb.
+```sh
+# Change permissions to avoid error "read only database"
+# chmod -R 766 /home/username/.pki/
```
Comment (pi-314159) [inline chromium/USAGE.md:31]: Could you change `/home/username` to `$HOME`?

Comment (nickforsythbarr) [reply chromium/USAGE.md:31]: needs to be run as sudo so not really.

Comment (pi-314159) [reply chromium/USAGE.md:31]: My bad. Everything looks good to me!

**Review by pi-314159 — APPROVED** [review]

**Comment by baentsch** [comment]
Thanks for the contribution!

**Pull request merged**